spec/classes/dev_spec.rb

```diff
@@ -77,7 +77,6 @@
         :osfamily               => 'Gentoo',
         :operatingsystem        => 'Gentoo',
         :operatingsystemrelease => '3.16.1-gentoo',
-        :concat_basedir         => '/dne',
         :is_pe                  => false,
         :concat_basedir         => '/foo',
```

Comment:
following all other `:concat_basedir` examples, this one is the correcter one to remove.